runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -98,6 +124,21 @@ void setTagString(int itemId, String tags, boolean variation)
 		}
 	}
 
+	public void setTagItemQuantity(String tag, int itemId, String input, boolean variation)
```

Comment:
I don't see where variations are actually used

Reply:
None were used, so it is now removed in the next commit.